e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-save-and-back.spec.js

```diff
@@ -164,11 +140,10 @@ describe('Insurance - Your business - Company details page - Save and go back',
     it('should render valid submitted values when going back to the page', () => {
       cy.navigateToUrl(url);
 
-      companyDetails.companiesHouseSearch().should('have.value', COMPANIES_HOUSE_NUMBER);
-      companyDetails.tradingNameYesRadioInput().should('be.checked');
-      companyDetails.tradingAddressYesRadioInput().should('be.checked');
-      companyDetails.phoneNumber().should('have.value', VALID_PHONE_NUMBERS.LANDLINE.NORMAL);
-      companyDetails.companyWebsite().should('have.value', WEBSITE_EXAMPLES.VALID);
+      companyDetails[TRADING_NAME].yesRadioInput().should('be.checked');
+      companyDetails[TRADING_ADDRESS].yesRadioInput().should('be.checked');
```

Comment:
Same as above, for all this file, whilst you're in here?

Reply:
done